digits/store/views.py

```diff
@@ -0,0 +1,125 @@
+# Copyright (c) 2014-2016, NVIDIA CORPORATION.  All rights reserved.
+from __future__ import absolute_import
+
+import flask
```

Comment:
Sorry to nitpick, but can you put this `import flask` below the standard imports? And above the digits imports?
https://www.python.org/dev/peps/pep-0008/#imports